roles/nvidia-dgx-firmware/defaults/main.yml

```diff
@@ -3,15 +3,22 @@
 # Which portions of the role to run
 run_diagnostics: true
 update_firmware: false
+load_firmware: false
```

Comment:
If we're adding a var to load the firmware, shouldn't we also make changes to the `nvidia-dgx-fw-update.yml` playbook and any associated docs?

Reply:
I'm changing the default behavior here to true and adding a comment.

This will simply copy the firmware container over and load it. We want to do this for both diagnostics and FW update, the only reason we would set this to false is if the container has already been loaded manually or we are re-running the playbook for some reason.